vimiv/gui/metadatawidget.py

```diff
@@ -54,20 +55,42 @@ def __init__(self, parent):
             self._mainwindow_bottom = 0
             self._mainwindow_width = 0
             self._path = ""
-            self._loaded = False
+            self._current_set = ""
 
             api.signals.new_image_opened.connect(self._on_image_opened)
+            api.settings.metadata.current_keyset.changed.connect(self._update_text)
 
             self.hide()
 
         @api.keybindings.register("i", "metadata", mode=api.modes.IMAGE)
         @api.commands.register(mode=api.modes.IMAGE)
-        def metadata(self):
-            """Toggle display of exif metadata of current image."""
+        def metadata(self, count: Optional[int] = None):
+            """Toggle display of exif metadata of current image and handle switching of metadata keysets.
+
+            **count:** Select the key set to display.
+            """
```

Comment:
Thanks for already working on the docstring! I would remove the "and handle..." from the summary though, and word the count as  "... to display instead.". This keeps it consistent with the slideshow documentation.

In principle this is the docstring I meant before where any user documentation can go. It is accessible via `:help :metadata` and in principle the website. This made me find a bug in the build gh-pages workflow which I have to fix :smile:

Reply:
Using the `:help` command, I can only access `metadata.current_keyset` but not `:metadata`.

Reply:
Is this also the case if you come from image mode?

Reply:
Oh I see. Works flawless in image mode